.travis.yml

```diff
@@ -35,15 +35,6 @@ deploy:
     on:
       tags: true
       condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+_[0-9]{3}$
-  - provider: releases
```

Comment:
Why stop publishing the yaml for draft releases?

Reply:
didnt find it useful